### PR TITLE
chore: bump deps

### PR DESCRIPTION
**Pull request by DmitriyMV**
- run rekres and fix nolint directives
- bump deps (keep gen to 0.4.8 for now) for server, client and tests

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -20,13 +20,13 @@ REGISTRY_AND_USERNAME ?= $(REGISTRY)/$(USERNAME)
 PROTOBUF_GRPC_GATEWAY_TS_VERSION ?= 1.2.1
 TESTPKGS ?= ./...
 NODE_BUILD_ARGS ?=
-PROTOBUF_GO_VERSION ?= 1.33.0
-GRPC_GO_VERSION ?= 1.3.0
-GRPC_GATEWAY_VERSION ?= 2.19.1
+PROTOBUF_GO_VERSION ?= 1.34.1
+GRPC_GO_VERSION ?= 1.4.0
+GRPC_GATEWAY_VERSION ?= 2.20.0
```
Comment (Unix4ever) [inline Makefile:25]: This Gateway version might have issues. But maybe it works if you bump the gateway version in the `go.mod`...

Please run `make generate generate-frontend` too.

Comment (DmitriyMV) [reply Makefile:25]: Done! Nothing changed.

Comment (utkuozdemir) [reply Makefile:25]: it doesn't work. need to change this line manually yo 2.19.1. then the `make generate-frontend && make generate`.

File: internal/pkg/grpcutil/logger_test.go [hunk 2]
```diff
@@ -74,7 +74,7 @@ func TestPayloadUnaryServerInterceptor(t *testing.T) {
 		errCh <- server.Serve(listener)
 	}()
 
-	dial, err := grpc.Dial(listener.Addr().String(),
+	dial, err := grpc.Dial(listener.Addr().String(), //nolint:staticcheck
```
Comment (smira) [inline internal/pkg/grpcutil/logger_test.go:77]: I feel we should fix these

**Review by smira — APPROVED** [review]

File: internal/pkg/grpcutil/logger_test.go (hunk 2 above)
Comment (Unix4ever) [reply internal/pkg/grpcutil/logger_test.go:77]: dial can just be replaced with `NewClient`

Comment (DmitriyMV) [reply internal/pkg/grpcutil/logger_test.go:77]: I'll do that in the separate PR.

**Pull request merged**